src/admin/components/Fields.vue

```diff
@@ -460,185 +479,207 @@
                 <p v-if="hasValidationError( fieldData.name )" class="dokan-error">
                     {{ getValidationErrorMessage( fieldData.name ) }}
                 </p>
-
-                <p class="description" v-html="fieldData.desc"></p>
-            </td>
+            </div>
         </template>
 
         <template v-if="'select' == fieldData.type">
-            <th scope="row">
-                <label :for="sectionId + '[' + fieldData.name + ']'">{{ fieldData.label }}</label>
-            </th>
-
-            <td v-bind:class="['combine' === fieldData.type ? 'tooltips-data combine-tips-style' : 'tooltips-data']">
-                <span v-if="fieldData.tooltip">
-                    <i class="dashicons dashicons-editor-help tips" :title="fieldData.tooltip" v-tooltip="fieldData.tooltip"></i>
-                </span>
-            </td>
-
-            <td v-bind:class="[fieldData.content_class ? fieldData.content_class : '']">
-                <select v-if="!fieldData.grouped" class="regular" :name="sectionId + '[' + fieldData.name + ']'" :id="sectionId + '[' + fieldData.name + ']'" v-model="fieldValue[fieldData.name]">
-                    <option v-if="fieldData.placeholder" value="" v-html="fieldData.placeholder"></option>
-                    <option v-for="( optionVal, optionKey ) in fieldData.options" :value="optionKey" v-html="optionVal"></option>
-                </select>
-
-                <select v-else class="regular" :name="sectionId + '[' + fieldData.name + ']'" :id="sectionId + '[' + fieldData.name + ']'" v-model="fieldValue[fieldData.name]">
-                    <option v-if="fieldData.placeholder" value="" disabled v-html="fieldData.placeholder"></option>
-                    <optgroup v-for="optionGroup in fieldData.options" :label="optionGroup.group_label">
-                        <option v-for="option in optionGroup.group_values" :value="option.value" v-html="option.label" />
-                    </optgroup>
-                </select>
+            <div class="field_contents" v-bind:class="[fieldData.content_class ? fieldData.content_class : '']">
+                <fieldset>
+                    <div class="field_data">
+                        <h3 class="field_heading" scope="row">
+                            {{ fieldData.label }}
+                        </h3>
+                        <p class="field_desc" v-html="fieldData.desc"></p>
+                    </div>
+                    <div class="field">
+                        <select v-if="!fieldData.grouped" class="regular" :name="sectionId + '[' + fieldData.name + ']'" :id="sectionId + '[' + fieldData.name + ']'" v-model="fieldValue[fieldData.name]">
+                            <option v-if="fieldData.placeholder" value="" v-html="fieldData.placeholder"></option>
+                            <option v-for="( optionVal, optionKey ) in fieldData.options" :value="optionKey" v-html="optionVal"></option>
```

Comment:
While using `v-for`, it's best practice to use a unique key binding. Ensure this for every `v-for`. For example, in this case,
```
<option v-for="( optionVal, optionKey ) in fieldData.options" :key="optionKey" :value="optionKey" v-html="optionVal"></option>
```